src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeReferenceNumberTruncatorTest.java

```diff
@@ -17,6 +17,7 @@
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.exceptions.RequiredFieldMissingException;
```

Comment:
The domain package shouldn't have any dependency/import on the infrastructure package -- can we move `RequiredFieldMissingException` into the domain?

Reply:
Sure

Reply:
Moved in the latest commit